src/pyskindose/table_data/generated_hvl_spekpy_data.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import spekpy as sp  # Note: Not included in PySkinDose,
+import pandas as pd
+import os
+from pathlib import Path
+
+
+def generate_spekpy_data(
+    kvpRange_kV,
+    InherentFiltration_mmAl,
```

Comment:
Should we change the Pascal casing to snake case for these input variables to adhere to standard python conventions?

Reply:
yes, we should. 

We should also do this to the whole project, but that affects a lot of code and should be covered in a separate branch

Reply:
done